src/AuditTrailModule/javasource/test_crm/tests/ExpectedLogLine.java

```diff
@@ -0,0 +1,34 @@
+
+package test_crm.tests;
+
+import static org.junit.Assert.*;
+
+import com.mendix.core.CoreException;
+
+import audittrail.proxies.LogLine;
+import audittrail.proxies.MemberType;
+
+public class ExpectedLogLine {
+	final MemberType memberType;
+	final String memberName;
+	final String oldValue;
+	final String newValue;
+
+	public ExpectedLogLine(final MemberType memberType, final String memberName, final String oldValue, final String newValue) {
+		this.memberType = memberType;
+		this.memberName = memberName;
+		this.oldValue = oldValue;
+		this.newValue = newValue;
+	}
+
+	public void assertEqual(final LogLine logLine) throws CoreException {
+		assertEquals("Incorrect membertype", this.memberType, logLine.getMemberType());
```

Comment:
Nitpick - There can be a space between member and type.

Reply:
Same as previous similar comment. Done.